.dockerignore

```diff
@@ -1 +1,2 @@
-*
+.git
+.gitignore
```

Comment:
we can put more things here- think about things we can exclude that are not relevant to prod